<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QWidget" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>500</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>521</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lePassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbPhone">
      <property name="text">
       <string>phone</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbEmail">
      <property name="text">
       <string>email</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbUsername">
      <property name="text">
       <string>username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbPassword">
      <property name="text">
       <string>password</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leUsername"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leEmail"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lePhone"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>570</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>RegisterForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>923</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
